user-service/src/main/java/rs/edu/raf/banka1/controller/AuthenticationController.java

```diff
@@ -50,7 +55,12 @@ public ResponseEntity<?> login(@RequestBody @Valid LoginRequest loginRequest) {
 
             UserResponse user = this.userService.findByEmail(loginRequest.getEmail());
             if (user == null) {
-                return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("User not found");
+                Customer customer = customerRepository.findCustomerByEmail(loginRequest.getEmail()).orElse(null);
+                if(customer == null) {
+                    return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("User not found");
```

Comment:
valjda je not found status code?

Reply:
Pre je isto bio UNAUTHORIZED, pa sam ostavio, mogu da promenim na not found

Reply:
Po meni ostavi unauthorized